### DISCUSSION TITLE
What is the most common surname in the 🌍?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#791] [Galaxy Trivia] [answered]
**Bonus:** Find the verb that contains only the letters of the first + second most common surnames.

**Comment by SANKARAMDAS** [comment]
More than 106 million people have the surname Wang, a Mandarin term for prince or king. It's not too surprising that the top surname is Chinese, as China has the world's largest population.

 J, M, S, D, and C are the most common initials for first surnames, whereas S, B, H, M, and C are the most common initials for last surnames.

**Comment by peterbaccaro** [comment, marked as answer]
More than 106 million people have the surname **Wang**, a Mandarin term for prince or king. It's not too surprising that the top surname is Chinese, as China has the world's largest population. As of 2022, it was home to 1.45 billion people, or 18.5 percent of the world's total.

The first and second most common surnames are "Wang" and "Li" To find a verb that contains only the letters of these surnames, we can use the letters from both surnames to form the verb "**wailing**".
